### ISSUE TITLE
need a fixed width for lights

### ISSUE THREAD

**Issue by mrix2000** [Prosk8er/Smashable-Christmas-Lights#1]
Hello and thanks for updating the smashable Christmas lights, I have been using the old mod for around 5 years, within your mod I notice the lights spread right across the top of the screen but on the old mod I was able to restrict them within my theme, I cannot for the life of me remember how that was done but I think I got advice from the orgional developer, would you know how your achieve this ? my website is here https://www.mrixadvertising.co.uk/
Basically I would like the light with the top blue section, I can understand they will be a lot smaller and closer together which was the case before but it looked great for my theme.
Huge thanks for any help and your time.
Best regards
mrix

**Comment by Prosk8er** [comment]
are you using phpbb 3.0.x or 3.1.x cause this is the mod for 3.0.x
the support for the validated extension for 3.1.x is on phpbb site
there is no support for phpbb 3.2.x at the moment since I have yet to fully test it
